src/config.rs

```diff
@@ -690,6 +691,7 @@ impl Default for TiKvConfig {
         TiKvConfig {
             log_level: LogLevelFilter::Info,
             log_file: "".to_owned(),
+            log_rotate_size: ReadableSize::mb(0),
```

Comment:
Please check the `log_rotate_size` in the `validate`, in case it rotates too fast.